### PR TITLE
tests:Update UI tests to pass with mock data

### PR DESCRIPTION
**Pull request by rsanteri**
### Requirements for a pull request

- [x] Unit tests related to the change have been updated 
- [ ] Documentation related to the change has been updated 

### Description of the Change

Update current full UI tests to pass with some mocked data. These tests were initially planned for e2e testing, but they are currently not run by any CI. We should consider if they are useful at all since UI have quite a coverage with component tests.

### CODE REVIEW

File: cypress/integration/debug.js [hunk 1]
```diff
@@ -1,75 +1,78 @@
...
+    cy.visit('/');
+    // navigate to Debug -view
+    cy.get('[data-testid="helpmenu-toggle"]')
+      .click()
+      .then(() => {
+        cy.get('[data-testid="helpmenu-link-debug"]').first().click();
+      });
+  });
+  
+  it('Debug - Test that feature flags are rendered', () => {
```
Comment (obgibson) [inline cypress/integration/debug.js:12]: You don't need to have the `Debug - ` in the test description. The `describe` description already has it. The output looks like 
```  Debug
    ✓ Debug - Test that feature flags are rendered (3591ms)
    ✓ Debug - Test network logging and discard logs (2417ms)
    ✓ Debug - Test network logging and download logs (1137ms)
```

I also like to have the description as the end of a sentence starting with "it". e.g. `it("renders feature flags")`

**Review by obgibson — APPROVED** [review]
Tests all pass now!

When I run `yarn test:components`. How do I get it to run all of the tests? I had to select each test file separately and run each one.  I am used to seeing a play icon to run them all.

File: cypress/integration/debug.js (hunk 1 above)
Comment (rsanteri) [reply cypress/integration/debug.js:12]: Agreed

**Comment by rsanteri** [comment]
> Tests all pass now!
> 
> When I run `yarn test:components`. How do I get it to run all of the tests? I had to select each test file separately and run each one. I am used to seeing a play icon to run them all.

Unfortunately cypress component tests doesn't have that at least yet. `yarn test:components` open interactive mode which is ideal when creating new tests or debugging single failing test. Use `yarn test:components:run` to run all tests in command line. (`yarn test` and `yarn test:ci` after #37)

**Pull request merged**